tests/kernel/fp_sharing/float_disable/src/k_float_disable.c

```diff
@@ -7,7 +7,7 @@
 #include <ztest.h>
 
 #define STACKSIZE 1024
-#define PRIORITY  5
+#define PRIORITY  0
```

Comment:
No particular need for this change AFAICT.  Though I have zero idea why "five!" seemed like a good priority originally either.

Reply:
It was arbitrary, @andyross , I think I copied from some other test.
Now , I was thinking to put 0, to make sure it looks "less arbitrary". I will make a comment that this does not really matter; all I need this for, is to have the two test threads with the same priority. I will also enhance the documentation inside the test.

Reply:
thanks.